rpc/types/src/json.rs

```diff
@@ -3,10 +3,7 @@
 //! <https://github.com/monero-project/monero/blob/cc73fe71162d564ffda8e549b79a350bca53c454/src/rpc/daemon_messages.h>.
 
 //---------------------------------------------------------------------------------------------------- Import
-use crate::{
-    base::{EmptyRequestBase, EmptyResponseBase, ResponseBase},
-    macros::define_request_and_response,
-};
+use crate::{base::ResponseBase, macros::define_request_and_response};
 
 //---------------------------------------------------------------------------------------------------- Struct definitions
 // This generates 2 structs:
```

Comment:
Any `{json,bin,other.rs}` changes can be ignored in this PR, they get implemented in the other PRs.